package.json

```diff
@@ -109,23 +109,23 @@
     "@types/yargs": "^17.0.10",
     "c8": "^9.0.0",
     "form-data": "^4.0.0",
-    "gapic-tools": "^0.4.0",
-    "gts": "^5.0.0",
+    "gapic-tools": "^0.4.6",
+    "gts": "^6.0.2",
     "jsdoc": "^4.0.0",
     "jsdoc-fresh": "^3.0.0",
     "jsdoc-region-tag": "^3.0.0",
-    "linkinator": "^3.0.0",
+    "linkinator": "^6.1.2",
     "mocha": "^9.2.2",
     "mockery": "^2.1.0",
+    "nise": "6.0.0",
```

Comment:
We can unpin this dep and path-to-regexp (unless you were going to do it in a separate PR?). See: https://github.com/googleapis/nodejs-storage/issues/2522

Reply:
I have removed those dependencies.